### PR TITLE
feat: Allow generic command definitions for flex workflows

### PR DESCRIPTION
**Pull request by alinamchin**
Allows commands to be defined in `service_config.yml` rather than having separate files per command.
Adds `flex list`  which lists available workflows.

https://flexport.atlassian.net/browse/DEVX-952

### CODE REVIEW

**Comment by u2mejc** [comment]
LGTM 👍

**Review by fp-mt — CHANGES_REQUESTED** [review]
This looks great!  I've got a few ideas to simplify, which we can do post-merge.

I just ran the tests locally though and they are failing, let's get those fixed before merging.  I'll sync up with @alinamchin on that.

File: flex/cmd/list.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package cmd
```
Comment (fp-mt) [inline flex/cmd/list.go:1]: New list feature needs a test.

Comment (alinamchin) [reply flex/cmd/list.go:1]: done!

**Review by fp-mt — APPROVED** [review]

**Pull request merged**